examples/git-credentials/git-push-job.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
```

Comment:
What's the purpose of this file?

Reply:
This job will push to the git-server which is running in a different pod.